<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cash_withdraw</class>
 <widget class="QDialog" name="cash_withdraw">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #05B893;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>170</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Norwester</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
text-align: centre;
font-size: 25px;
</string>
   </property>
   <property name="text">
    <string>ENTER AMOUNT TO WITHDRAW:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_funds">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>220</y>
     <width>291</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Norwester</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: white;
 border-width: 10px;
 border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="username_ID">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Norwester</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #007871;
font-size: 15px;
</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_done">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>440</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Norwester</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #F54646;
color: white;
 border-width: 5px;
 border-radius:7px;
</string>
   </property>
   <property name="text">
    <string>DONE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>639</x>
     <y>440</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Norwester</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #007871;
color: white;
 border-width: 5px;
 border-radius:7px;
</string>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_multiplesof500">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>310</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Norwester</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: white;
font-size: 15px;
</string>
   </property>
   <property name="text">
    <string>(MULTIPLES OF 500)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_funds_warning">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>240</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Myriad Hebrew</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: red;
font-size: 13px;
border-bottom: 1px red solid;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
